examples/async_tasks/async_compute.rs

```diff
@@ -76,12 +78,12 @@ fn spawn_tasks(mut commands: Commands, thread_pool: Res<AsyncComputeTaskPool>) {
 /// removes the task component from the entity.
 fn handle_tasks(
     mut commands: Commands,
-    mut transform_tasks: Query<(Entity, &mut Task<Transform>)>,
+    mut transform_tasks: Query<(Entity, &ComputeTransformTask)>,
     box_mesh_handle: Res<BoxMeshHandle>,
     box_material_handle: Res<BoxMaterialHandle>,
 ) {
-    for (entity, mut task) in transform_tasks.iter_mut() {
-        if let Some(transform) = future::block_on(future::poll_once(&mut *task)) {
+    for (entity, compute_transform_task) in transform_tasks.iter_mut() {
+        if let Some(transform) = compute_transform_task.0.poll() {
```

Comment:
I think we should derive `Deref` on `ComputeTransformTask`, so you don't need this `.0.`

Reply:
Irrelevant after rebase, since I did not need to change the example.